pg_partitioning/decorators.py

```diff
@@ -2,14 +2,16 @@
 from typing import Type
 
 from django.db import models
+
 from pg_partitioning.manager import ListPartitionManager, TimeRangePartitionManager
 
 logger = logging.getLogger(__name__)
 
 
 class _PartitioningBase:
-    def __init__(self, partition_key: str, **options):
```

Comment:
could you add some document to describe when we should ignore the check

Reply:
@virusdefender any updates?